src/models/tenant.cr

```diff
@@ -121,6 +122,15 @@ class Tenant
     add_error("credentials", e.message.to_s)
   end
 
+  # Try parsing the JSON for booking limits to test that it works
+  private def validate_booking_limits
+    if booking_limits_column.defined?
+      Hash(String, Int32).from_json booking_limits.to_json
```

Comment:
```suggestion
      Hash(String, Int32).from_json(booking_limits.to_json)
```

Extremely minor style suggestion, you can ignore/accept at your leisure.
The main reason I prefer parens is that it provides a clearer delineation of where the arguments are.
Another reason is that parens make it easier to come along and chain another method.